Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
+    NSDecimalCopy(&decimal, &_decimal);
+    return decimal;
+}
+
+- (NSDecimalNumber*)_decimalOperation:(_DecimalOperations)operand
+                        decimalNumber:(NSDecimalNumber*)decimalNumber
+                         withBehavior:(id<NSDecimalNumberBehaviors>)behavior {
+    RETURN_NULL_IF(!decimalNumber);
+
+    NSDecimal a = [self decimalValue];
+    NSDecimal b = [decimalNumber decimalValue];
+
+    NSDecimal result;
+    NSCalculationError error = NSCalculationNoError;
+
+    switch (operand) {
+        case _Addition:
+            error = NSDecimalAdd(&result, &a, &b, [behavior roundingMode]);
+            break;
+        case _Subtraction:
+            error = NSDecimalSubtract(&result, &a, &b, [behavior roundingMode]);
+            break;
+        // case _Multiplication:
+        //    error = NSDecimalMultiply(&result, &a, &b, [behavior roundingMode]);
+        //   break;
+        //  case _Division:
+        //  error = NSDecimalDivide(&result, &a, &b, [behavior roundingMode]);
+        // break;
```

Comment:
it's dead code now, with change to function pointer.

---
In reply to: [80787909](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r80787909) [](ancestors = 80787909)